docs/cloud/tutorials/helm.md

```diff
@@ -548,27 +502,26 @@ appVersion: "1.16.0"
 * `templates/deployment.yaml`:
```

Comment:
To upgrade the helm chart. Command working for me is:

$ helm upgrade test-name example

Reply:
where is this mentioned? Could make the comment under the line that you are talking about?

Reply:
Line 365